src/utils/getFormattedNumber.ts

```diff
@@ -14,7 +18,7 @@ class NumberParser {
     this._numeral = new RegExp(`[${numerals.join('')}]`, 'g');
     this._index = (d) => index.get(d);
   }
-  parse(string) {
+  parse(string: string) {
```

Comment:
The type defined here is fine, but I see a warning while using `parse`

Argument of type 'number' is not assignable to parameter of type 'string'.